src/components/AddBlockButton/AddBlockButton.stories.tsx

```diff
@@ -14,12 +15,18 @@ export default {
     tags: ['autodocs'],
     args: {
         title: 'Add Block Button',
+        direction: ButtonDirection.Right,
         orientation: AddBlockButtonDirection.Horizontal,
```

Comment:
You can remove the orientation from the Storybook args.
We deprecate it so no need to document it. We just don't remove from the component yet.